File: 1547.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        while (n-- > 0) {
            int QT = sc.nextInt();
            int S = sc.nextInt();

            int pos = 0;
            int menor = 1000;

            for (int j = 0; j < QT; j++) {
                int guess = sc.nextInt();
                int diff = Math.abs(S - guess);

                if (diff < menor) {
                    menor = diff;
                    pos = j + 1;
                }
            }

            System.out.println(pos);
        }

        sc.close();
    }
}

File: 1547.py
n = int(input())

for i in range(n):
    QT, S = map(int, input().split())

    menor = 1000
    pos = 0
    quesses = list(map(int, input().split()))
    for j, quess in enumerate(quesses):
        diff = abs(S - quess)
        if diff < menor:
            menor = diff
            pos = j + 1

    print(pos)

